### PR TITLE
re-work ref impl to use path based routing instead

### PR DESCRIPTION
**Pull request by nabuskey**
Updated it to use path routing instead and added support for running it in web browsers through Codespaces. 

During testing, I noticed the default minimum spec given by Codespaces was not enough to run all the examples. I've updated the default Codespaces spec to 4 cores. 

In some environments, the proxy address provided by GitHub may be blocked on host machines due to security configurations provided by organizations. In that case, you cannot run and use this (or any apps) in a web browser. They will need to run it through the GH CLI or locally.

### CODE REVIEW

File: examples/ref-implementation/README.md [hunk 1]
```diff
@@ -49,15 +54,15 @@ The only package that cannot be removed this way is Keycloak because other packa
 
 
 #### Accessing UIs
-- Argo CD: https://argocd.cnoe.localtest.me:8443
-- Argo Workflows: https://argo.cnoe.localtest.me:8443
-- Backstage: https://backstage.cnoe.localtest.me:8443
-- Gitea: https://gitea.cnoe.localtest.me:8443
-- Keycloak: https://keycloak.cnoe.localtest.me:8443/admin/master/console/
+- Argo CD: https://cnoe.localtest.me:8443/argocd
+- Argo Workflows: https://cnoe.localtest.me:8443/argo
```
Comment (blakeromano) [inline examples/ref-implementation/README.md:58]: just for clarity here maybe `/workflows` would be easier to find

**Review by csantanapr — APPROVED** [review]
LGTM

File: examples/ref-implementation/README.md (hunk 1 above)
Comment (csantanapr) [reply examples/ref-implementation/README.md:58]: +1 /workflows or /argo-workflows

**Review by greghaynes — APPROVED** [review]

File: examples/ref-implementation/README.md (hunk 1 above)
Comment (nabuskey) [reply examples/ref-implementation/README.md:58]: renamed to argo-workflows

**Pull request merged**